src/view/com/lightbox/ImageViewing/index.tsx

```diff
@@ -232,6 +327,48 @@ function LightboxFooter({
   )
 }
 
+function interpolateTransform(
```

Comment:
this little goblin took me several days to get right but it looks obvious in retrospect